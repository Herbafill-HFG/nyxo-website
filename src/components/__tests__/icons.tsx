import React from "react"
import renderer from "react-test-renderer"

import { Icon } from "../Icons"

describe("Icons", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Icon name="sun" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
