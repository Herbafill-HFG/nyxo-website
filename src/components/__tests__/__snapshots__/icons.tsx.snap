// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons renders correctly 1`] = `
<svg
  className="sc-bdnylx LxIqb"
  fill="none"
  height="50px"
  stroke="#333"
  strokeLinecap="round"
  strokeWidth="1.5"
  version="1.1"
  viewBox="0 0 30 30"
  width="50px"
  xmlns="http://www.w3.org/2000/svg"
>
  <g>
    <path
      d="M 12,7.5c2.485,0,4.5,2.015,4.5,4.5s-2.015,4.5-4.5,4.5S7.5,14.485,7.5,12S9.515,7.5,12,7.5z "
    />
    <path
      d="M 12,0.75V4.5 "
    />
    <path
      d="M 12,19.5v3.75 "
    />
    <path
      d="M 23.25,12H19.5 "
    />
    <path
      d="M 4.5,12H0.75 "
    />
    <path
      d="M 20.25,3.75l-3,3 "
    />
    <path
      d="M 6.75,17.25l-3,3 "
    />
    <path
      d="M 20.25,20.25l-3-3 "
    />
    <path
      d="M 6.75,6.75l-3-3"
    />
  </g>
</svg>
`;
